test/testcafe/framework/page-objects/BasePageObject.js

```diff
@@ -100,17 +102,22 @@ export default class BasePageObject {
     return this.form.getElement(ionMessagesSelector).innerText;
   }
 
+  getSignoutLinkElement() {
+    const selector = userVariables.v3 ? BACK_TO_SIGNIN_LINK : SIGNOUT_LINK;
+    return Selector(selector);
```

Comment:
Yeah, so that's actually the sort of redundancy we can eliminate when we encounter it and use a more direct/targeted selector that does some of the assertion work for us implicitly (since we are asking for the exact piece of text we don't need to also check if the text is that text we asked for, as you pointed out).

Would be great if we have a `getLink(text)` method in BaseForm/BasePage and then the {*}Page classes could wrap those with `getSignoutLink` that calls `getLink("Signout Link Text")` or something like that.

It's up to you if you think deprecation + new method is the right move or if updating all the references in the tests to use a new method is right. I think the latter is best if there are only a few or zero instances of the method outside of the scope of this ticket.